src/locale.js

```diff
@@ -5,6 +5,7 @@ import formatSpecifier from "./formatSpecifier";
 import formatTypes from "./formatTypes";
 import {prefixExponent} from "./formatPrefixAuto";
 import identity from "./identity";
+import trimInsignificantZeros from "./trim";
```

Comment:
Let’s rename trim.js to trimZeros.js, and in both cases where we import the function, say

```js
import trimZeros from "./trimZeros";
```

To avoid a collision we can still use `trim` as the local variable in this file.

Reply:
On second thought, I’m just going to make a few changes directly to this PR. Please hold.